### PR TITLE
Work Staff List- Add Tooltips

### PR DESCRIPTION
**Pull request by djnunez-aot**
#2176 

-Add tooltips to all text so that full name will display on overflow

### CODE REVIEW

File: epictrack-web/src/components/work/workStaff/WorkStaffList.tsx [hunk 1]
```diff
@@ -75,6 +80,21 @@ const WorkStaffList = () => {
               .join(", ")}`,
           enableHiding: false,
           enableColumnFilter: true,
+          Cell: ({ row, renderedCellValue }) => {
+            const staff = row.original.staff
+              .filter((p: { role: { name: any } }) => p.role.name === rolename)
+              .map(
+                (p: { first_name: string; last_name: string }) =>
```
Comment (dinesh-aot) [inline epictrack-web/src/components/work/workStaff/WorkStaffList.tsx]: Can you make it lastname, firstname and join with "; "

File: epictrack-web/src/components/work/workStaff/WorkStaffList.tsx [hunk 2]
```diff
@@ -165,12 +196,35 @@ const WorkStaffList = () => {
         header: "Responsible EPD",
         filterVariant: "multi-select",
         filterSelectOptions: responsibleEpdFilter,
+        Cell: ({ row, renderedCellValue }) => {
+          const { responsible_epd } = row.original;
+          return (
+            <ETParagraph
+              enableTooltip
```
Comment (dinesh-aot) [inline epictrack-web/src/components/work/workStaff/WorkStaffList.tsx]: Would you need tooltip for responsible EPD. It is going to be single person?

File: epictrack-web/src/components/work/workStaff/WorkStaffList.tsx [hunk 3]
```diff
@@ -165,12 +196,35 @@ const WorkStaffList = () => {
...
       },
       {
         accessorKey: "eao_team.name",
         header: "Team",
         filterVariant: "multi-select",
         filterSelectOptions: teamFilter,
+        Cell: ({ row, renderedCellValue }) => {
+          return (
+            <ETParagraph
+              enableTooltip
```
Comment (dinesh-aot) [inline epictrack-web/src/components/work/workStaff/WorkStaffList.tsx]: same here, team name is very short and I don't think tooltip is needed.

File: epictrack-web/src/components/work/workStaff/WorkStaffList.tsx [hunk 4]
```diff
@@ -70,11 +75,26 @@ const WorkStaffList = () => {
...
-                  `${p.first_name} ${p.last_name}`
+                  `${p.last_name} ${p.first_name}`
               )
-              .join(", ")}`,
+              .join("; ")}`,
           enableHiding: false,
           enableColumnFilter: true,
+          Cell: ({ row, renderedCellValue }) => {
+            const staff = row.original.staff
+              .filter((p: { role: { name: any } }) => p.role.name === rolename)
```
Comment (jadmnsaad) [inline epictrack-web/src/components/work/workStaff/WorkStaffList.tsx]: please don't use single letter variables

File: epictrack-web/src/components/work/workStaff/WorkStaffList.tsx [hunk 5]
```diff
@@ -165,12 +196,18 @@ const WorkStaffList = () => {
         header: "Responsible EPD",
         filterVariant: "multi-select",
         filterSelectOptions: responsibleEpdFilter,
+        Cell: ({ renderedCellValue }) => {
+          return renderedCellValue;
```
Comment (jadmnsaad) [inline epictrack-web/src/components/work/workStaff/WorkStaffList.tsx]: what is this doing? is it not redundant?

Comment (djnunez-aot) [reply epictrack-web/src/components/work/workStaff/WorkStaffList.tsx]: removed it

File: epictrack-web/src/components/work/workStaff/WorkStaffList.tsx (hunk 4 above)
Comment (djnunez-aot) [reply epictrack-web/src/components/work/workStaff/WorkStaffList.tsx]: changed all the variables should be more readable now

**Review by jadmnsaad — APPROVED** [review]

**Pull request merged**